crates/client/src/account/register.rs

```diff
@@ -41,10 +49,55 @@ pub fn register() -> Html {
                 placeholder="Username"
                 name="username"
                 id="username"
-                type="username"
+                type="text"
             />
             <button type="submit">{ "Register" }</button>
         </form>
         </>
     }
 }
+
+fn hash_password(password: &str) -> String {
+    let mut hasher = Sha256::new();
+    hasher.update(password);
+
+    format!("{:x}", hasher.finalize())
+}
+
+#[derive(Serialize)]
+struct User {
+    username: String,
+    email: String,
+    password_hash: String,
+}
+
+async fn register_in_backend(
+    hashed_password: &str,
+    username: &str,
+    email: &str,
+) -> Result<(), reqwest::Error> {
+    let user = User {
```

Comment:
Nothing is wrong here, but serde have json! macro which can be used in cases like this :)

Reply:
thanks